dom.bs

```diff
@@ -3822,87 +3822,55 @@ the empty string instead, and then do as described below, switching on <a>contex
  <a>nodes</a>.
 </dl>
 
+<p>The <dfn export>contiguous exclusive {{Text}} nodes</dfn> of a <a>node</a> <var>node</var> are
```

Comment:
Maybe put this and the "exclusive Text node" concept right after wholeText, or in some way together and close to the "contiguous Text nodes" concept? It should help make it obvious without scrolling around what the difference between these things are. (I was first going to suggest cross-linked notes, that'd be another way.)